packages/client/src/StreamVideoClient.ts

```diff
@@ -303,8 +304,6 @@ export class StreamVideoClient {
         this.writeableStateStore.registerCall(theCall);
       }),
     );
-
-    return connectUserResponse;
```

Comment:
This is potentially breaking. Can you revert this change back?

Reply:
Yes, it's a mistake that it was removed. Thanks.